scout/server/blueprints/variants/controllers.py

```diff
@@ -44,6 +39,24 @@ def populate_chrom_choices(form, case_obj):
     form.chrom.choices = [("", "All")] + [(chrom, chrom) for chrom in chromosomes]
 
 
+def check_missing_evaluation_terms(store):
```

Comment:
Does not seem to be working properly.

If I do the following the `cust000/643595/variants?variant_type=clinical` view is crashing.

1. load default terms
2. goto random SNV variant for a case
3. annotate variant as "Known Pathogenic"
4. load a custom-terms file without the tag "Known Pathogenic"
5. goto or reload variants list view

Reply:
Fixed, flashing this message instead:

![image](https://user-images.githubusercontent.com/28093618/124250624-57e79600-db25-11eb-8adf-d7aba520b81f.png)